src/main/java/com/besmartkinopoiskservice/service/impl/MovieServiceImpl.java

```diff
@@ -48,7 +48,7 @@ public MovieDetailsResponseTO findMovie(UUID movieId) throws ServiceException {
         if (!movie.isPresent()){
             throw new ServiceException("Ошибка при поиске фильма");
```

Comment:
Можно вот так красиво разрулить)
```suggestion
    movie = movieRepository.find...orElseThrow(() => new ServiceException(...));
```